miqa/core/rest/experiment.py

```diff
@@ -70,7 +72,7 @@ class ExperimentViewSet(ReadOnlyModelViewSet, mixins.CreateModelMixin):
     def get_queryset(self):
         projects = get_objects_for_user(
             self.request.user,
-            [f'core.{perm}' for perm in Project.get_read_permission_groups()],
+            [f'core.{perm}' for perm in Project().get_read_permission_groups()],
```

Comment:
Here, `self` is not an `Experiment` object, but an `ExperimentViewSet` object. I would try to call `self.get_object` here but `get_queryset` is called before that and would cause recursion.